### PR TITLE
[CHAT-561] Add empty sample

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
# Purpose

Add Empty sample which should be like the Blank sample but without Conversation 1 and Conversation 2

### CODE REVIEW

File: Empty/en/cognitive-flow.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {
+          "type": "message",
+          "messages": [
+            "This is a welcome conversation for your chatbot."
+          ]
+        },
+        {
+          "type": "conversation",
+          "conversation": "help",
+          "conditions": [
```
Comment (tsvetie) [inline Empty/en/cognitive-flow.json]: Remove condition

Comment (tsvetie) [reply Empty/en/cognitive-flow.json]: Checked with Georgi - we need this

File: Empty/ar/cognitive-flow.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  },
+  "settings": {
+    "invalid-replies": [
+      "لست متأكداً من أنني فهمت ما قلته."
+    ],
+    "general-failure": [
+      "نحن نواجه صعوبات تقنية في هذه اللحظة."
+    ],
+    "previous-conversation-messages": [
+      "سأعود الآن إلى {{conversation DisplayName}} الآن."
```
Comment (tsvetie) [inline Empty/ar/cognitive-flow.json]: conversationDisplayName

Comment (TsvetanMilanov) [reply Empty/ar/cognitive-flow.json]: Done.

**Pull request merged**